lib/server.js

```diff
@@ -47,7 +47,10 @@ function Server(options, isSecure, onListening) {
           else {
             xml = Serializer.serializeMethodResponse(value)
           }
-          response.writeHead(200, {'Content-Type': 'text/xml'})
+          response.writeHead(200, {
+            'Content-Type': 'text/xml',
+            'Content-Length': xml.length
```

Comment:
Should this instead use `Buffer.byteLength(xml, 'utf8')` instead of `xml.length`?